pkg/controllers/monitoring-stack/alertmanager.go

```diff
@@ -20,7 +20,7 @@ func newAlertmanager(
 	if resourceSelector == nil {
 		resourceSelector = &metav1.LabelSelector{}
 	}
-	replicas := int32(3)
+	replicas := int32(2)
```

Comment:
We are making the same change for CMO: https://github.com/openshift/cluster-monitoring-operator/pull/1489